docs/image.rst

```diff
@@ -0,0 +1,108 @@
+Image
+*****
+
+.. py:module:: microbit
+
+Classes
+=======
+
+.. py:class:: Image(width_or_string=None, height=None, buffer=None)
+
+    If ``width_or_string`` is a string, creates an image from that string.
+
+    If it's an integer, a height has to also be specified, and it creates
+    an image with ``width_or_string`` columns and ``height`` rows. Optionally
+    ``buffer`` can be an array of ``width_or_string``×``height`` integers to
+    initialize the image.
```

Comment:
I'm not sure about from_string being the default. I think the default should be an empty image with optional dimensions (defaulting to 5×5) -- that seems like the most basic thing. Then you could have:

``Image.from_string(string)``
``Image.from_data(width, height, data, mono=False)``

and also ``image.load_data(data, mono=False)`` for changing an existing image.